editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -655,6 +687,97 @@ describe('interactionUpdate with stacked strategy change', () => {
   })
 })
 
+describe('interactionUpdate with accumulating keypresses', () => {
+  it('steps an interaction session correctly', () => {
+    let interactionSession = createInteractionViaMouse(
+      canvasPoint({ x: 100, y: 200 }),
+      { alt: false, shift: false, ctrl: false, cmd: false },
+      { type: 'BOUNDING_AREA', target: EP.elementPath([['aaa']]) },
+    )
+    if (interactionSession.interactionData.type === 'KEYBOARD') {
```

Comment:
you create the interactionSession with `createInteractionViaMouse`, so it looks like this IF will never be true